scripts/quests/otherAreas/A_Generous_General.lua

```diff
@@ -0,0 +1,290 @@
+-----------------------------------
+-- A Generous General
+-----------------------------------
+-- Log ID: 4, Quest ID: 109
+-----------------------------------
+-- Oldton zone !pos 566 -12 685 106
+-- Faulpie     !gotoid 17719379
+-- iron box    !pos -141 7 157 11
+-----------------------------------
+local oldtonID = zones[xi.zone.OLDTON_MOVALPOLOS]
+local southernSanDoriaID = zones[xi.zone.SOUTHERN_SAN_DORIA]
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.A_GENEROUS_GENERAL)
+
+local spawnedMobs =
```

Comment:
This seems like we could procedurally generate this as opposed to defining a table, but would need to look into the function you're using to call this

Reply:
Something like

```
for i = 0, 6 do
    table.insert(spawnedMobs, oldtonID.mob.GENEROUS_GENERAL_OFFSET + i)
end
```

I just saw other methods defining tables so I assumed that was the accepted pattern

Reply:
I think my issue here is that its only used once.  I'm fine with it, but feel like the spawning logic could be simplified overall and iterated